src/browsers/stories/heatmap/Heatmap.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { scaleBand, scaleSqrt, scaleSequential, interpolateRgb, extent } from 'd3'
+import DotplotHeatmap from '../../tob/shared/components/DotplotHeatmap'
+import { defaultCellTypeColors } from '../../tob/shared/utilities/constants'
+
+export const Heatmap = () => {
+  const columnNames = Object.keys(defaultCellTypeColors())
+  const rowNames = ['Gene1', 'Gene2', 'Gene3']
+  const data = Array.from({ length: 42 }, (_, i) => ({
```

Comment:
Currently, since the user passes in a height (or uses the default one), the heatmap won't resize its height with a  changing number of genes. Might need to change that in future to maybe set height as number of genes * some constant for row height. Or make it scrollable?

Reply:
Made variable height with number of genes

Reply:
Cool! Setting a variable height depending on the number of genes is the right way to go